### ISSUE TITLE
Email Archive Add Archive Job throws error when attempting to add job and access to SMB share is denied

### ISSUE THREAD

**Issue by deeztek** [deeztek/Hermes-Secure-Email-Gateway#3]
Following error is displayed when attempting to add Email Archive Job and access to SMB share is denied:

```
Lucee 5.2.9.31 Error (application)
Message	Error invoking external process
Detail	/bin/cp: cannot create regular file '/mnt/hermesarchivetest/testsmb': Permission denied
Stacktrace	The Error Occurred in
/var/www/html/admin/email_archive.cfm: line 595
593: outputfile ="/dev/null"
594: arguments="-inputformat none">
595: </cfexecute>
596:
597:
```

Need to update cfcatch for "permission denied" in error detail.

**Comment by deeztek** [comment]
Will address in version 2.0 template.

**Comment by deeztek** [comment]
Fixed on #25
